### ISSUE TITLE
Add-VSTeamClassificationNode which allows adding iterations and areas

### ISSUE THREAD

**Issue by smholvoet** [DarqueWarrior/vsteam#276]
Am I missing something or is there no `Add-VSTeamClassificationNode` function, which can add iterations or areas? I did find the [Get-VSTeamClassificationNode](https://github.com/DarqueWarrior/vsteam/blob/master/Source/Public/Get-VSTeamClassificationNode.ps1) which uses the same URI, but there doesn't seem to be a function which adds classificationNodes (iterations or areas)....? 🤔 

See [Classification Nodes - Create Or Update](https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/classification%20nodes/create%20or%20update?view=azure-devops-rest-5.1)

I'd be willing to start a PR in order to add this functionality, unless I somehow missed a function which actually does the same.

**Comment by MichelZ** [comment]
You did not miss that function.

**Comment by SanderHolvoet** [comment]
Ok, I consider this to be something which other people mind find useful. Especially if you have multiple  teams contained within a single team project who each need their own area and respective iterations.

I'll make sure to reference this issue upon creation of my PR.

**Comment by SebastianSchuetze** [comment]
@SanderHolvoet if you find a API part that is note covered yet with a cmdlet, then you can add it if you want. 
Go ahead. Read the contribution docs and make a PR. ☺️

**Comment by SanderHolvoet** [comment]
Had a look at the API and I feel like splitting this up into two separate cmdlets would make more sense:

- Add-VSTeamArea
- Add-VSTeamIteration

Especially since these 2 types of classification nodes require additional parameters in their respective request body.

**Area:**
```json
{
  "name": "Web"
}
```

**Iteration:**
```json
{
  "name": "Final Iteration",
  "attributes": {
    "startDate": "2014-10-27T00:00:00Z",
    "finishDate": "2014-10-31T00:00:00Z"
  }
}
```

The official Azure DevOps API uses the [TreeStructureGroup](https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/classification%20nodes/get?view=azure-devops-rest-5.1#treestructuregroup) to make the distinction between either _areas_ or _iterations_. 
Perhaps we could also split up the current `Get-VSTeamClassificationNode` into two seperate cmdlets in order to keep things uniform with the 2 _Add_ cmdlets mentioned above.

This obviously diverts from the API who uses the "classification nodes" terminology, but I suspect this would make things easier for people who are looking for a cmdlet which has "area" or "iteration" in its name (speaking from my own experience). The Azure DevOps UI also uses areas and iterations as opposed to classification nodes.

Would appreciate some guidance before I continue and invest more time in this.

**Comment by MichelZ** [comment]
It is a good idea to split up.
Altough... I personally would still create an `Add-VSTeamClassificationNode`, and call this from `Add-VSTeamArea` / `Add-VSTeamIteration`. (Same with the Get-*)

I wanted to add `Get-VSTeamIteration` before but forgot (there is an issue #152 for this)

**Comment by SebastianSchuetze** [comment]
@SanderHolvoet with everything we added here it is good to start with the terminology that we are used with. The API is not the most intuitive one. So I would suggest to first create the two split cmdlets. True, they are grouped into the classification nodes, but I would suggest to create to seperate ones.

And since @MichelZ already created already the one for Get-VSTeamClassificationNode. It would be good to have one like Add-VSTeamClassificationNode and the two split ones use this Add-VSTeamClassificationNode one.

Long story short: I agree with MichelZ on this. Same would work for Get-*

**Comment by SebastianSchuetze** [comment]
@smholvoet and @MichelZ this has been added with the last PR and will be available with the next release.
